daprdocs/content/en/operations/performance-and-scalability/perf-actors-activation.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Actors activation performance"
+linkTitle: "Actors activation performance"
+weight: 10000
+description: ""
+---
+This article provides service invocation API performance benchmarks and resource utilization for Actor in Dapr on Kubernetes. 
+
+## System overview
+
+For applications using Actors in Dapr there are two aspects to be considered. First, is the routing of actor invocations handled by Dapr sidecar. Second, is the Actor runtime that is implemented and handled on the application side and depends on the SDK. For now, the performance tests are using the Java SDK to provide an Actor runtime in the application.
+
+### Kubernetes components
+
+* Sidecar (data plane)
+* Placement (required for Actors, control plane mapping ActorType to hosts)
+* Operator (control plane)
+* Sidecar Injector (control plane)
+* Sentry (optional, control plane)
+
+For more information see [overview of Dapr on Kubernetes]({{< ref kubernetes-overview.md >}}).
+
+## Performance summary for Dapr v1.0
+
+The Actors API in Dapr sidecar will identify which hosts are registered for a given Actor type and route the request to the appropriate host for a given Actor Id. The host runs an instance of the application and uses the Dapr SDK (.Net, Java, Python or PHP) to handle Actor requests via HTTP.
```

Comment:
Id --> ID

Reply:
It is hard, but you need to minimize the use of "will" and use present tense as much as possible "will identify which hosts" reword to "The actors API in Dapr sidecar identifies which hosts are registered..." as in this is what it does today.